server/app/controllers/api/v1/suggestqueries_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class SuggestqueriesController < ApplicationController
```

Comment:
How about `SuggestQueriesController`

Reply:
I prefer to use suggestqueries to express this is on word.

Reply:
I also agree SuggestQueriesController. 

Because `Suggest` and `Queries` are different words, and `suggestqueries` is also not a terminology.

Reply:
`Api::V1::Exceptions::SuggestQueryKeyInvalidError` not same ....

Reply:
ok change to SuggestQueriesController, API url will change to /suggest_queries, I will update the doc.